Userland/Applications/KeyboardSettings/KeyboardSettingsWidget.cpp

```diff
@@ -309,6 +309,10 @@ void KeyboardSettingsWidget::set_keymaps(Vector<DeprecatedString> const& keymaps
 
 void KeyboardSettingsWidget::write_caps_lock_to_ctrl_sys_variable(bool caps_lock_to_ctrl)
 {
+    if (getuid() != 0) {
+        return;
+    }
```

Comment:
No curly braces

Reply:
Sounds good. Thank you for your patience!